.gitignore

```diff
@@ -19,5 +19,7 @@ yarn-error.log*
 *.sln
 *.sw?
 
+*.iml
```

Comment:
Ik geloof best dat dit erin moet. Maar vraag me gwn af wat er nu een .iml file maakt

Reply:
.iml is een bestand van intellij. Volgens mij een settings fo een workspace bestand